modules/tftp/tftp_plugin.rb

```diff
@@ -2,6 +2,8 @@ module Proxy::TFTP
   class Plugin < ::Proxy::Plugin
     plugin :tftp, ::Proxy::VERSION
 
+    capability :bootloader_universe
```

Comment:
Are we still using this?

Reply:
Yes, the capability is used in Foreman to determine weather to access the files in the host specific directory or to use the old default location in `:tftproot:/grub2`. With this we can guarantee the backward compatibility of Foreman X.Y to Smart proxy X.(Y-1).